library/src/androidTest/java/org/xmtp/android/library/ClientTest.kt

```diff
@@ -85,6 +89,7 @@ class ClientTest {
     }
 
     @Test
+    @Ignore("CI Issues")
```

Comment:
I'm not sure why is using this `@Ignore`

Reply:
as you can see [here](https://github.com/xmtp/xmtp-android/actions/runs/7103203135/job/19335347623#step:9:441) we can not run a local environment on CI. is the reason why most of the test that requires create a Client are ignored

Reply:
You actually can run in the local environment in CI. IOS is the only repo who has that issue. You can see more of how this is setup here https://github.com/xmtp/xmtp-android/blob/main/.github/workflows/test.yml#L51

Reply:
so why was the test pass in my local and it didn’t in CI? 🤔

Reply:
@kele-leanes can you link me to the CI that failed?

Reply:
https://github.com/xmtp/xmtp-android/actions/runs/7106161017/job/19344969828 It seems there is an error creating the client